src/poliastro/plotting/porkchop.py

```diff
@@ -1,6 +1,3 @@
-"""
-This is the script for porkchop plotting
-"""
 import numpy as np
```

Comment:
Let us add here a new module docstring better than the original :memo:

Reply:
Any suggestions for the same?

Reply:
Did some changes. Please let me know anything else is out of place :smile: